cairo/tests/utils/strategies.py

```diff
@@ -58,6 +58,9 @@
     st.one_of(st.binary(), uint, st.text(), st.booleans()), st.lists
 )
 
+# Custom types
+stack = st.lists(uint256)
```

Comment:
you need to register it below in `register_type_strategies` but don't need this `stack = ...` here